src/tests/json.cxx

```diff
@@ -0,0 +1,105 @@
+#include "../json.h"
+#include "../logger.h"
+#include <gtest/gtest.h>
+
+using namespace rapidjson;
+
+TEST(json, merge_01) {
```

Comment:
I advise against this style of numbered unit test names. This is because explicit names for what you are testing make it easy for someone else to see what the code does. The test itself ensures that the code does what you intend, and the descriptive name makes it easy for someone else to spot the other possibility of a bug: that the intent of the programmer is flawed.
This test, for example, could be called `check_element_in_second_document_replaces_element_in_first_document_with_same_key`